sky/skylet/LICENCE

```diff
@@ -203,16 +203,16 @@
 --------------------------------------------------------------------------------
 
 Code in providers/azure from
-https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/_private/_azure
-Git Revision: 7f1bacc7dc9caf6d0ec042e39499bbf1d9a7d065
+https://github.com/ray-project/ray/tree/ray-1.13.0/python/ray/autoscaler/_private/_azure
+Git Revision: 4ce38d001dbbe09cd21c497fedd03d692b2be3e
 
 Code in providers/gcp from
-https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/_private/gcp
-Git Revision: ef9d9df4e7454d428a958281e9de333795dccb44
+https://github.com/ray-project/ray/tree/ray-1.13.0/python/ray/autoscaler/_private/gcp
+Git Revision: 4ce38d001dbbe09cd21c497fedd03d692b2be3e
```

Comment:
Are these hashes correct? 

https://github.com/ray-project/ray/blob/4ce38d001dbbe09cd21c497fedd03d692b2be3e/python/ray/autoscaler/_private/gcp/config.py

returns 404. I checked that `gcp/config.py` latest commit in the 1.13 branch is 6d83a3f2832fc24f63af1c1270afdd4850d701db.

Reply:
Ahhh, I put the latest git commit for the tag ray-1.13.0. Let me fix it.